module/VuFind/src/VuFind/OAI/Server.php

```diff
@@ -497,6 +540,16 @@ protected function initializeSettings(\Zend\Config\Config $config)
         if (isset($config->OAI->set_query)) {
             $this->setQueries = $config->OAI->set_query->toArray();
         }
+
+        if (!empty($config->OAI->vufind_api_format_fields)) {
+            $this->vufindApiFields = explode(
+                ',', $config->OAI->vufind_api_format_fields
+            );
+            $this->metadataFormats['oai_vufind'] = [
+                'schema' => 'http://vufind.org/oai_vufind-6.0.xsd',
```

Comment:
Is the 6.0 here because you're assuming this will be released as part of VuFind 6.0? Probably a reasonable assumption due to the backward-breaking constructor signature change. Just making sure I understand what you're doing!

If you do want to create an xsd document to serve at this URL, I could certainly make that happen... though all things being equal, it might be better to use a directory, like `https://vufind.org/xsd/oai_vufind-6.0.xsd`.

Reply:
Yes, I'm hoping this would be part of 6.0. And those paths were just a suggestion. I'll see if I can manage to create a suitable xsd.

Reply:
Then again, maybe the format could be 1.0 too. Not sure which one is better.